main.nf

```diff
@@ -613,27 +615,127 @@ process dia_spectral_library_search {
+	-extraction_function ${params.extraction_function} \\
+	-outer_loop_threads ${params.outer_loop_threads} \\
+	-ms1_isotopes ${params.ms1_isotopes} \\
+	-no_progress ${params.no_progress} \\
+	-Calibration:ms1_im_calibration ${params.ms1_im_calibration} \\
+	-Calibration:im_correction_function ${params.im_correction_function} \\
+	-Calibration:debug_im_file ${params.debug_im_file} \\
+	-Calibration:debug_mz_file ${params.debug_mz_file} \\
+        -Library:retentionTimeInterpretation ${params.retentionTimeInterpretation} \\
+	-Library:override_group_label_check ${params.override_group_label_check} \\
+	-Library:force_invalid_mods ${params.force_invalid_mods} \\
+	-min_upper_edge_dist ${params.min_upper_edge_dist} \\
+	-sonar ${params.sonar} \\
         -RTNormalization:alignmentMethod ${params.irt_alignment_method} \\
         -RTNormalization:estimateBestPeptides \\
         -RTNormalization:outlierMethod none \\
+	-RTNormalization:useIterativeChauvenet ${params.useIterativeChauvenet} \\
+	-RTNormalization:RANSACMaxIterations ${params.ransacMaxIterations} \\
+        -RTNormalization:RANSACMaxPercentRTThreshold ${params.ransacMaxPercentRTThreshold} \\
+        -RTNormalization:RANSACSamplingSize ${params.ransacSamplingSize} \\
+        -RTNormalization:InitialQualityCutoff ${params.initialQualityCutoff} \\
+        -RTNormalization:OverallQualityCutoff ${params.overallQualityCutoff} \\
+	-RTNormalization:MinPeptidesPerBin ${params.minPeptidesPerBin} \\
         -RTNormalization:NrRTBins ${params.irt_n_bins} \\
         -RTNormalization:MinBinsFilled ${params.irt_min_bins_covered} \\
+	-RTNormalization:lowess:span ${params.span} \\
+        -RTNormalization:b_spline:num_nodes ${params.num_nodes} \\
         -mz_correction_function quadratic_regression_delta_ppm \\
         -Scoring:stop_report_after_feature 5 \\
+        -Scoring:rt_normalization_factor ${params.rt_normalization_factor} \\
+        -Scoring:quantification_cutoff ${params.quantification_cutoff} \\
+        -Scoring:write_convex_hull ${params.write_convex_hull} \\
+        -Scoring:spectrum_addition_method ${params.spectrum_addition_method} \\
+        -Scoring:add_up_spectra ${params.add_up_spectra} \\
+        -Scoring:spacing_for_spectra_resampling ${params.spacing_for_spectra_resampling} \\
+        -Scoring:uis_threshold_sn ${params.uis_threshold_sn} \\
+        -Scoring:uis_threshold_peak_area ${params.uis_threshold_peak_area} \\
+        -Scoring:scoring_model ${params.scoring_model} \\
+        -Scoring:im_extra_drift ${params.im_extra_drift} \\
+        #-Scoring:strict ${params.strict} \\
+        -Scoring:TransitionGroupPicker:stop_after_feature ${params.stop_after_feature} \\
+        -Scoring:TransitionGroupPicker:min_peak_width ${params.min_peak_width} \\
+        -mz_correction_function quadratic_regression_delta_ppm \\
+        -Scoring:stop_report_after_feature 5 \\
+	-Scoring:rt_normalization_factor ${params.rt_normalization_factor} \\ ??
```

Comment:
What's with the "??"? This just breaks the command. You want clear shell newlines here.

Reply:
Okay, I removed those.